apps/src/convert.cpp

```diff
@@ -88,12 +77,12 @@ int main(int argc, char** argv)
             continue;
         }
 
-        image = ToneMap(image, 2.2);
+        image = et::ToneMap(image, 2.2);
 
         image *= MAX_INTENSITY;
         image.convertTo(image, CV_16UC1);
```

Comment:
This line and the multiply above will need to change based on what output depth the user specifies.